### PR TITLE
Next tag

### PR DESCRIPTION
**Pull request by nemchik**

### CODE REVIEW

**Comment by nemchik** [comment]
I've commented out the nightly tag in the readme vars for the moment. I have been unable to get https://github.com/linuxserver/docker-thelounge/pull/38 working correctly. This tag works as expected and can be released.

File: Dockerfile [hunk 1]
```diff
@@ -5,36 +5,45 @@ ARG BUILD_DATE
...
- npm install -g \
-	thelounge@${THELOUNGE_VERSION} && \
- echo "**** ensure public true on startup aka no users ****" && \
- sed -i "s/public: false,/public: true,/g" /usr/lib/node_modules/thelounge/defaults/config.js && \
- echo "**** cleanup ****" && \
- rm -rf \
-	/root && \
- mkdir -p / \
-	/root
+	echo "**** install build packages ****" && \
```
Comment (TheSpad) [inline Dockerfile:15]: These are all tabs

**Review by Roxedus — CHANGES_REQUESTED** [review]
Dockerfile formating needs updating

File: Dockerfile (hunk 1 above)
Comment (nemchik) [reply Dockerfile]: Resolved in https://github.com/linuxserver/docker-thelounge/pull/37/commits/b0388d0dcc15b932aa2e336e0acb90fdf3d4e547

**Comment by nemchik** [comment]
> Dockerfile formating needs updating

Resolved in https://github.com/linuxserver/docker-thelounge/pull/37/commits/b0388d0dcc15b932aa2e336e0acb90fdf3d4e547

**Review by TheSpad — APPROVED** [review]
LGTM

**Pull request merged**